engine/src/main/java/org/camunda/bpm/engine/task/TaskQuery.java

```diff
@@ -1104,4 +1104,9 @@ public interface TaskQuery extends Query<TaskQuery, Task> {
    *                                this exception, {@link #or()} must be invoked first.
    */
   TaskQuery endOr();
+
+  /**
+   * Evaluates existence of attachment and comments associated with the task, defaults to false.
+   */
+  TaskQuery withCommentAttachmentInfo();
```

Comment:
Note to myself:
* authorizations don't exist for comments and attachments so we won't need to cover them.